### DISCUSSION TITLE
Re-licensing the project

### DISCUSSION THREAD

**Discussion by NTBBloodbath** [rest-nvim/rest.nvim#272] [Polls]
Hey, I hope everyone is doing well.

I was thinking about changing the license of `rest.nvim` to [GPLv3](https://www.gnu.org/licenses/gpl-3.0.en.html), as a user this shouldn't worry you so much so this is aimed at potential and existing contributors. The reason is that I really hate working with MIT software unless it is really necessary, since the GPL is a copyleft license that guarantees end users the four freedoms to run, study, share, and modify the software.

However, I know there are people who might not like this, so I'd rather hold a vote before just changing things because that's how it seems to me.

**Comment by simanga-dev** [comment]
I am just curious, What do you hate about MIT?

**Reply by NTBBloodbath** [reply]
MIT is one of the most permissive licenses out there, you don't even have to give credits if you yoink some code from a MIT licensed software and that sucks. To give some more context, MIT is so permissive that any large company _or even individuals_ can exploit your open-source code for commercial purposes without even acknowledging it, completely free.
